config-controller/api/v1alpha1/policy_types.go

```diff
@@ -38,31 +38,47 @@ type EnforcementAction string
 type SecurityPolicySpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^[^\n\r\$]{5,128}$`
-	PolicyName string `json:"policyName,omitempty"`
+	// PolicyName is the name of the policy as it appears in Central.  Note that changing this value will rename the policy in Central.  This field must be unique.
```

Comment:
nit: "... as it appears in Central" is a little vague. Maybe we should hash out a standard way to refer to things like this.

Not worth changing just for this, but here's a brainstorm we could consider in the future:
```suggestion
	// PolicyName is the name of the policy as it appears in the API and UI.  Note that changing this value will rename the policy as stored in the database.  This field must be unique.
```